app.js

```diff
@@ -57,6 +66,44 @@ var server = app.listen(port, function () {
     console.log('Server running at http://127.0.0.1:' + port + '/');
 });
 
+//initial meeting endpoints
+
+//create a meeting
+app.post("/meeting", function (req, res) {
```

Comment:
The [ticket](https://github.com/zednis/alexa-skill-meeting-scheduler/issues/9) suggests the POST handle a JSON payload that corresponds to the following structure:

```json
{
  "name": "example meeting",
  "startDateTime": "2018-02-24T12:00:00Z",
  "endDateTime": "2018-02-24T13:00:00Z"
}
```

From my understanding of the documentation, we can use the bodyParser library (already referenced) to get a parsed req.body based on the JSON request payload.

```js
var name = req.body.name || null; 
var start_datetime = req.body.startDateTime || null;
var end_datetime = req.body.endDateTime || null;
var room = req.body.endDateTime || null;
```

PS - the ``var x = y || null`` is used to ensure the value of x is set to null if y is undefined.

Reply:
Fixed this

Reply:
Looks good.